data/sql_incremental_aggregates/prepare_schedule_a_aggregate_size.sql

```diff
@@ -33,6 +33,10 @@ create index on ofec_sched_a_aggregate_size_tmp (size);
 create index on ofec_sched_a_aggregate_size_tmp (total);
 create index on ofec_sched_a_aggregate_size_tmp (count);
 
+-- this drops totals during rebuild
+drop table if exists ofec_sched_a_aggregate_state cascade;
+drop table if exists ofec_sched_a_aggregate_size_old cascade;
```

Comment:
Would it make sense to remove the `alter table` statements below this if we're just going to drop them wholesale here?